htdocs/stripe/class/stripe.class.php

```diff
@@ -367,18 +367,20 @@ public function createPaymentStripe($amount, $currency, $origin, $item, $source,
 			$order = new Commande($this->db);
 			$order->fetch($item);
 			$ref = $order->ref;
-			$description = "ORD=" . $ref . ".CUS=" . $societe->id;
+			$description = "ORD=" . $ref . ".CUS=" . $societe->id.".PM=stripe";
 		} elseif ($origin == invoice) {
 			$invoice = new Facture($this->db);
 			$invoice->fetch($item);
 			$ref = $invoice->ref;
-			$description = "INV=" . $ref . ".CUS=" . $societe->id;
+			$description = "INV=" . $ref . ".CUS=" . $societe->id.".PM=stripe";
 		}
 
 		$metadata = array(
 			"dol_id" => "" . $item . "",
 			"dol_type" => "" . $origin . "",
 			"dol_thirdparty_id" => "" . $societe->id . "",
+      "FULLTAG" => $description,
```

Comment:
Why adding FULLTAG ? We already have "dol_type" (saying it is order or invoice) and "dol_id" giving id of order or invoice.

Reply:
because page with list of payment use it but we can change this other page and pass.